src/main/webapp/electionlist.html

```diff
@@ -94,10 +94,16 @@ <h3>Elections coming up in {{state}}</h3>
             {{#each stateElections}}
             <li class="list-group-item election-item">
               <div class="election-quick-details">
-                <p>{{this.name}}</p>
+                <div id="state-election-name">
+                  <p>{{this.name}}</p>
+                </div>
                 <p><em>{{formatDate this.date}}</em></p>
               </div>
-              <div class="learn-more-button">Learn more</div>
+              <a href="#" 
+                 onclick="redirectToStateElectionPage({{{this.ID}}});return false;" 
```

Comment:
I'm not that  familiar with JS, do we need to have the "return false" bit?

Reply:
This keeps the page from redirecting should there be an issue with the first function call (https://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-a-click-event-listener).